mass_mailing_event_registration_exclude/models/mailing.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import copy
+
+from odoo import fields, models
+
+
+def event_filtered_ids(model, mailing_mailing_id, domain, field="email"):
```

Comment:
It doesn't make much sense to keep this as a global method. Let's put the logic inside the mailing model